packages/ui/src/helpers/authenticator/utils.ts

```diff
@@ -4,9 +4,14 @@
  */
 
 import { Hub } from 'aws-amplify';
+import { appendToCognitoUserAgent } from '@aws-amplify/auth';
 import { AuthInterpreter, HubHandler } from '../../types';
 import { ALLOWED_SPECIAL_CHARACTERS } from './constants';
 
+// Re-exporting this helper method so the public packages
+// don't need to import anything from '@aws-amplify/auth'
+export { appendToCognitoUserAgent };
```

Comment:
Nitty, but maybe we can wrap this in a "configure" util in case we want to add other functionality in the feature outside appending to the user agent. Example:
```
type ConfigureOptions = { packageName: string; version: string }
const configureComponent = ({ packageName, version }: ConfigureOptions) => {
  // "@aws-amplify/ui-react" + "/" + "3.5.10"
  appendToCognitoUserAgent(`${packageName}/${version}`);
)

Reply:
oh yea thats a good idea!